core/src/autogluon/core/metrics/__init__.py

```diff
@@ -354,6 +354,14 @@ def rmse_func(predictions, targets):
                       greater_is_better=True,
                       needs_threshold=True)
 
+roc_auc_ovo_macro = make_scorer('roc_auc_ovo_macro',
+                                sklearn.metrics.roc_auc_score,
+                                multi_class='ovo',
+                                average='macro',
```

Comment:
Is macro the only variant that matters?

Reply:
There are other options available, but with macro it makes it insensitive to class imbalance.

https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html